src/main/js/wiki/containers/Settings.tsx

```diff
@@ -74,11 +80,40 @@ const Settings: FC<Props> = (props) => {
           onClick={() => {
             editWikiMutation.mutate({ landingPage: landingPage, rootDir: rootDir });
           }}
+          disabled={
```

Comment:
It would be great if the same behaviour is applied to the input that contains the error. Currently, the user has absolutely no idea that any path is invalid. Maybe changing the blue border of the input field to the red accent color from the Cloudogu theme would be nice.